test/PreAMMBatcher-e2e.ts

```diff
@@ -19,8 +18,6 @@ import {
   switchTokenTestInput,
 } from "./resources/testExamples";
 
-use(solidity);
```

Comment:
I never really knew what this was. Great to know we don't need it anymore.

Reply:
This would add the `chai` solidity machers (`expect(...).to.be.revertedWith(...)` for example). This gets automatically done by `buidler` now.